core/primitives/src/runtime/config_store.rs

```diff
@@ -349,7 +355,13 @@ mod tests {
         for version in testnet_store.store.keys() {
             let snapshot_name = format!("testnet_{version}.json");
             let config_view = RuntimeConfigView::from(store.get_config(*version).as_ref().clone());
-            insta::assert_json_snapshot!(snapshot_name, config_view);
+            any_failure |= std::panic::catch_unwind(|| {
+                insta::assert_json_snapshot!(snapshot_name, config_view);
+            })
+            .is_err();
+        }
+        if any_failure {
+            panic!("some snapshot assertions failed");
```

Comment:
`assert_snapshot` will terminate the singular test, however since there are multiple such assertions in the same test, those other assertions will not run until the previous failing one is fixed. In the past that required to run the test suite and `cargo insta` to update snapshots multiple times over.